lib/MusicBrainz/Server/Report/NotVariousArtists.pm

```diff
@@ -0,0 +1,47 @@
+package MusicBrainz::Server::Report::NotVariousArtists;
+use Moose;
+use MusicBrainz::Server::Constants qw( $VARTIST_ID );
+
+with 'MusicBrainz::Server::Report::RecordingReport',
+     'MusicBrainz::Server::Report::FilterForEditor::RecordingID';
+
+sub query {
+    "
+        SELECT
+            r.id AS recording_id,
+            row_number() OVER (ORDER BY r.artist_credit, r.name)
```

Comment:
It's using the `row_number()` window function to assign a sort order to each row: http://www.postgresql.org/docs/8.4/static/functions-window.html

I assume this is because the actual ordering parameters (r.artist_credit, r.name) are not stored for later use.